README.md

```diff
@@ -1,5 +1,6 @@
 <div align="center">
   <h1>javascript-utilities</h1>
+  <h3>:warning: This repo is in maintenance mode. Every new feature should be add to <a href="https://github.com/shopify/quilt">@shopify/quilt</a> :warning:</h3>
```

Comment:
```suggestion
  <h3>:warning: This repo is in maintenance mode. New features should be add to [@shopify/quilt](https://github.com/Shopify/quilt). Also see [this issue](https://github.com/Shopify/quilt/issues/136) for a rough plan on how old features can be migrated over to quilt. :warning:</h3>
```

Reply:
I can't put Markdown in HTML 😞

Reply:
oh funky there is html inside .md file.
either way its the content suggestion that is important. I will update it to HTML format.